src/extras/BitmapText.js

```diff
@@ -503,10 +503,10 @@ export default class BitmapText extends core.Container
      *
      * @static
      * @param {XMLDocument} xml - The XML document data.
-     * @param {PIXI.Texture} texture - Texture with all symbols.
+     * @param {PIXI.Texture[]} textures - List of textures for each page.
```

Comment:
That's wrong.

According to your code this can be either texture or an array of textures.

`PIXI.Texture|PIXI.Texture[]`